industry_benchmarks/utils/fix_networks.py

```diff
@@ -666,6 +681,13 @@ def parse_args(arg_list = None):
         help="Results JSON file(s)",
         required=True,
     )
+    parser.add_argument(
+        "--allow-missing",
+        help="If we should ignore partially complete or missing edges and fix the network "
+             "rather than fail by default this is `False` and will raise an error.",
+        action="store_true",
```

Comment:
I'm not that familiar with argparse and just noticed that the `store_true` would actually already result in False as default?

Reply:
Thats how I also have it that `store_true` implies a default of False but I think there is no harm in setting it as well.